src/Components/Avatar.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+export function Avatar({ src, size, alt, children, color, variant }) {
+  let fSize = 20;
```

Comment:
Please rename variable fSize to something contextual like fontSize here.

Reply:
Changed it to fontSize.